src/pet.js

```diff
@@ -0,0 +1,94 @@
+const START_AGE = 0;
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_HUNGER = 0;
+const CRITICAL_FITNESS = 3;
+const CRITICAL_HUNGER = 5;
+const PET_DEAD_ERROR = 'Your pet is no longer alive :(';
+
+function Pet(name) {
+  this.name = name;
+  this.age = START_AGE;
+  this.hunger = MINIMUM_HUNGER;
+  this.fitness = MAXIMUM_FITNESS;
+  this.children = [];
+};
+
+Pet.prototype = {
+  get isAlive() {
+    return this.age < 30 && this.hunger < 10 && this.fitness >= 0;
+  }
+};
+
+Pet.prototype.growUp = function () {
+  if (!this.isAlive) {
+    throw new Error(PET_DEAD_ERROR);
+  };
+
+  this.age += 1;
+  this.hunger += 5;
+  this.fitness -= 3;
+};
+
```

Comment:
Same as above I would define a couple of constants for this: AGE_INCREMENT, etc.

Reply:
And then, once you define its  increments/decrements, you can replace all of those magic numbers below.